docs/root/start/quick-start/_include/envoy-demo.yaml

```diff
@@ -52,7 +52,6 @@ static_resources:
         sni: www.envoyproxy.io
 
 admin:
-  access_log_path: /dev/null
```

Comment:
with these ones in the docs - and esp in the getting started section we will need to check the literalinclude lines